.github/workflows/repository-scanner.yml

```diff
@@ -44,11 +52,18 @@ jobs:
   bidi-scanner:
     runs-on: ubuntu-latest
     steps:
-
       - name: Checkout the repo to Scan
-        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846  # v3.0.0
+        uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # v3.2.0
+
+      - name: Checkout the Actions repo
+        uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # v3.2.0
+        with:
+          repository: stephenfuqua/Ed-Fi-Actions
+          path: Ed-Fi-Actions
 
       - name: Bidirectional Trojan source detector
-        uses: haveyoudebuggedit/trojansourcedetector@103b76ee743f7a666fd98dfaa15b4b7e314ca140 # v1.1.0
+        uses: ./Ed-Fi-Actions/bidi-scanner
```

Comment:
This step is failing: https://github.com/stephenfuqua/Ed-Fi-Actions/actions/runs/3897441569/jobs/6655130894#step:4:4